src/foam/nanos/iban/ValidationIBAN.js

```diff
@@ -38,13 +38,13 @@ Columns: validation format, parsing format, example`,
           ["CRFF FFFF FFFF FFFF FFFF FF", "", "CR72 0123 0000 0171 5490 15"],
           ["CYFF FFFF FFFF AAAA AAAA AAAA AAAA", "CYkk bbbs ssss cccc cccc cccc cccc", "CY17 0020 0128 0000 0012 0052 7600"],
           ["CZFF FFFF FFFF FFFF FFFF FFFF", "", "CZ65 0800 0000 1920 0014 5399"],
-          ["DEFF FFFF FFFF FFFF FFFF FF", "DEkk bbbb bbbb cccc cccc cc", "DE89 3704 0044 0532 0130 00"],
+          ["DEFF FFFF FFFF FFFF FFFF FF", "DEkk ssss ssss cccc cccc cc", "DE89 3704 0044 0532 0130 00"],
           ["DKFF FFFF FFFF FFFF FF", "", "DK50 0040 0440 1162 43"],
           ["DOFF UUUU FFFF FFFF FFFF FFFF FFFF", "", "DO28 BAGR 0000 0001 2124 5361 1324"],
-          ["EEFF FFFF FFFF FFFF FFFF", "EEkk bbss cccc cccc cccx", "EE38 2200 2210 2014 5685"],
+          ["EEFF FFFF FFFF FFFF FFFF", "EEkk bbss cccc cccc cccc", "EE38 2200 2210 2014 5685"],
           ["EGFF FFFF FFFF FFFF FFFF FFFF FFFF F", "", "EG80 0002 0001 5678 9012 3451 8000 2"],
           ["ESFF FFFF FFFF FFFF FFFF FFFF", "ESkk bbbb ssss xxcc cccc cccc", "ES91 2100 0418 4502 0005 1332"],
-          ["FIFF FFFF FFFF FFFF FF", "FIkk bbbb bbcc cccc cx", "FI21 1234 5600 0007 85"],
+          ["FIFF FFFF FFFF FFFF FF", "FIkk ssss sscc cccc cc", "FI21 1234 5600 0007 85"],
```

Comment:
Why change the 'x' to 'c'?

Reply:
Assuming 'x' is not included in the account number and the Finland account number is 8 digits. https://bank.codes/iban/structure/finland/.

Reply:
From wikipedia, Finland iban format is "FIkk bbbb bbcc cccc cx" (the last digit is a check digit) but https://bank.codes/iban/structure/finland/ suggested the account number field should be 8 digits ie. "FIkk ssss sscc cccc cc". 

Should we follow wiki or bank.codes? @jlhughes @kgrgreer

Reply:
x is included in the account number. It just isn't included because you commented out line 516.